examples/jupyter/jupyter.json

```diff
@@ -0,0 +1,24 @@
+{
+	"cpus": 1, 
```

Comment:
Does this need a whole CPU and 512mb memory? Might be easier to deploy if this requirement is slackened.